extract_pipeline.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pandas as pd
+
+response = requests.get('https://randomuser.me/api/')
```

Comment:
Please turn your entire API request to a function and please you need to raise error just in case the request didn't work.